packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx

```diff
@@ -127,18 +127,36 @@ export const Docs: () => JSX.Element = () => {
         Controls](./?path=/docs/composites-call-basicexample--basic-example#customize-call-controls) to learn more.
       </Description>
 
-      <Heading>Adding file sharing</Heading>
+      <Heading>Adding file sharing in ACS Chat</Heading>
       <SingleLineBetaBanner />
       <Description>
-        The CallWithChat Composite supports file sharing capabilities in conjunction with your choice of a storage
-        solution. Using the provided APIs, you can enable the composite to support uploading files and displaying them
-        on the send box before sending, and the message thread once they have been sent or received. For an end to end
-        tutorial on enabling file sharing with Azure Blob Storage as your choice of a storage solution, refer to our
-        [tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/file-sharing-tutorial). File
-        sharing is supported for Teams interop scenarios by default.
+        In a Chat thread where participants are the Communication users, The `CallWithChat` Composite supports file
+        sharing capabilities in conjunction with your choice of a storage solution. Using the provided APIs, you can
+        enable the composite to support uploading files and displaying them on the send box before sending, and the
+        message thread once they have been sent or received. For an end to end tutorial on enabling file sharing with
+        Azure Blob Storage as your choice of a storage solution, refer to our
+        [tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/file-sharing-tutorial).
       </Description>
       <Source code={addFileSharingSnippet} />
 
+      <Heading>Adding file sharing in Teams Interop Chat</Heading>
+      <SingleLineBetaBanner />
+      <Description>
+        When using the `CallWithChat` Composite in a Teams interop scenario, the ACS user can recieve file attachments
```

Comment:
You are using two different terminologies and it can be confusing. In the above case you refer to ACS users as "Communication users" and here you are using the term ACS user. You want to try to align to using ACS users (I think thats what we do in our SDK documentation as well? Might need to double check)

Reply:
right thanks!